getting_started/first_3d_game/06.jump_and_squash.rst

```diff
@@ -344,6 +346,10 @@ destroy the mob.
         QueueFree();
     }
 
+.. note::
+
+    When using C#, Godot will create the appropriate events automatically for all Signals ending with `EventHandler` :ref:`C# Signals <doc_c_sharp_signals>`
```

Comment:
Just a nit-pick, but I think this sounds like there can be signals with delegates that don't end with `EventHandler` (but that would result in error [GD0201](https://docs.godotengine.org/en/latest/tutorials/scripting/c_sharp/diagnostics/GD0201.html)). And, technically, the generated event **is** the signal, the delegate is just its type.

But I do think it's important to mention that events are automatically generated because a lot of people miss it, thanks for adding a note about it.

Reply:
Oh! Makes sense! I'll try to find a better wording since I'll have to fix some lint tasks anyway